tests/infra/remote.py

```diff
@@ -278,13 +281,19 @@ def pid(self):
         return self._pid
 
     def suspend(self):
+        if not self.is_ccf_node:
```

Comment:
I'm not convinced it's worth adding all this extra stuff and the flag just in testing infra on the off chance someone tries to suspend a client.

Can we just get the call to pid() out of the ctor if the command doesn't start with "./cchost"?